tensorflow/compiler/xla/service/gpu/target_util.h

```diff
@@ -40,15 +44,31 @@ enum class TargetIntrinsicID {
   kBlockIdy,
   kBlockIdz,
   kBarrierId,
+  kPow,
+  kErfcinv,
+  kLog,
+  kLog1p,
+  kSin,
+  kCos,
+  kExp,
+  kExpm1,
+  kSqrt,
+  kRsqrt,
+  kAtan2,
+  kFmod,
+  kRound,
 };
 
-// Emits a call to the specified target intrinsic with the given operands.
+// Emits a call to the specified target function  with the given operands.
+// Target function can either be an intrinsic or a device function.
```

Comment:
Add a blank line (i.e. containing only "//") after this sentence to indicate a new paragraph.